### PR TITLE
feature: Expand scaffolded files to include a rollup build configurat…

### PR DESCRIPTION
**Pull request by tristanmenzel**
…ion when using --platform node

- Drop tsconfig package in favour of extending @tsconfig/nodeXX
- Add option to copy-package-json command to read from config. This version attempts to be a bit smarter (and opinionated) in how it crafts the final package.json
- Include audit script for all platforms

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+build
+# don't lint nyc coverage output
+coverage
+# don't lint generated types
+**/generated/types.d.ts
+**/generated/types.ts
+# don't lint ide files
+.idea
+.vscode
+src/templates
```
Comment (robdmoore) [inline .eslintignore:16]: What's this?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -50,14 +50,6 @@ jobs:
           access: 'public'
           check-version: true
 
-      - name: Publish @makerx/ts-config (if version has been updated)
```
Comment (robdmoore) [inline .github/workflows/release.yml:53]: Is this not needed?

File: scripts/dog-food-init.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { init } from '../src/init'
```
Comment (robdmoore) [inline scripts/dog-food-init.ts:1]: Nice

File: .eslintignore (hunk 1 above)
Comment (robdmoore) [reply .eslintignore:16]: Ahhh I see

**Review by robdmoore — APPROVED** [review]

File: rollup.config.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import nodeResolve from '@rollup/plugin-node-resolve'
+import typescript from '@rollup/plugin-typescript'
+import json from '@rollup/plugin-json'
+import type { RollupOptions } from 'rollup'
+import pkg from './package.json' with { type: 'json' }
```
Comment (pleb) [inline rollup.config.ts:5]: Today I learn

File: .github/workflows/release.yml (hunk 2 above)
Comment (tristanmenzel) [reply .github/workflows/release.yml:53]: > Drop tsconfig package in favour of extending @tsconfig/nodeXX

File: .eslintignore (hunk 1 above)
Comment (tristanmenzel) [reply .eslintignore:16]: I think it's not needed - originally the eslint script was using `src/**`, changing to `src/**/*.ts` fixed the issue I was having (see previous build failures)

**Pull request merged**